cmd/cli/version_test.go

```diff
@@ -180,37 +161,28 @@ func TestOutputVersionInfo(t *testing.T) {
 					},
 				},
 			},
-			expected:   "CLI Version: version.Info{Version:\"v0.0.0\", GitCommit:\"xxxxxxx\", BuildDate:\"0000-00-00-00:00\"}\nMesh [test] Version: version.Info{Version:\"v0.0.0\", GitCommit:\"xxxxxxx\", BuildDate:\"0000-00-00-00:00\"}\n",
+			expected:   "\nMESH NAME\tMESH NAMESPACE\tVERSION\tGIT COMMIT\tBUILD DATE\ntest\t\tv0.0.0\txxxxxxx\t0000-00-00-00:00\n",
 			clientOnly: false,
 		},
 		{
-			name: "cli and mesh versions with no remote version info",
-			versionInfo: versionInfo{
-				cliVersionInfo: &version.Info{
-					Version:   "v0.0.0",
-					GitCommit: "xxxxxxx",
-					BuildDate: "0000-00-00-00:00",
-				},
-				remoteVersionInfo: nil,
-			},
-			expected:   "CLI Version: version.Info{Version:\"v0.0.0\", GitCommit:\"xxxxxxx\", BuildDate:\"0000-00-00-00:00\"}\n",
-			clientOnly: false,
+			name:                  "cli and mesh versions with no remote version info",
```

Comment:
I'd rename both of these tests from "cli and mesh versions..." to "mesh versions..." since `outputPrettyVersionInfo` doesn't touch the client version at all anymore.